### PR TITLE
Matmul benchmark

### PR DESCRIPTION
**Pull request by cailun01**
### What changes were proposed in this pull request?

Provided a more accurate measurement of the matrix multiplication duration.

### Why are the changes needed?

Accurate measurements of matrix multiplication times can provide evidence for detecting stragglers.

### CODE REVIEW

**Review by workingloong — APPROVED** [review]
LGTM

File: dlrover/trainer/torch/node_check/ascend_npu.py [hunk 1]
```diff
@@ -53,9 +54,19 @@ def main():
     if use_cuda:
         local_rank = int(os.environ["LOCAL_RANK"])
         torch.cuda.set_device(local_rank)
+        if local_rank == 0:
```
Comment (samplise) [inline dlrover/trainer/torch/node_check/ascend_npu.py:57]: please add the description of the check method at the beginning of the main function. The same to nvidia_gpu.py

Comment (cailun01) [reply dlrover/trainer/torch/node_check/ascend_npu.py:59]: descriptions added.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**